### DISCUSSION TITLE
formily  builder  数据源和响应器的配置弹出框不能拖拽

### DISCUSSION THREAD

**Discussion by wsxe9988** [alibaba/formily#2006] [Ideas]
![1628821958863_CC2F3470-4D9C-474f-A425-8F1A7C6AB408](https://user-images.githubusercontent.com/5828057/129297761-21ab390f-6c80-4234-96db-b829085c9a1c.png)

**Comment by GadflyX** [comment]
你好，我是 formily 的用户，在引入表单设计器遇到一些问题，方便的话，想问你一些问题？
